packages/plugins/test/mocha/foundry/utils.spec.ts

```diff
@@ -209,7 +209,7 @@ describe('Utils', async () => {
       // Fifth contract:
       expect(
         existsSync(
-          `${artifactFolder}/test/deeper/DuplicateContractName.sol/DuplicateContractName.json`
+          `${artifactFolder}/deep/deeper/DuplicateContractName.sol/DuplicateContractName.json`
```

Comment:
This is a rider. One of our tests was failing because Foundry made a change that caused this artifact to be output in a different directory. I believe they fixed a bug with this change, and it does not appear to affect us outside of this test so I just updated the test.